### DISCUSSION TITLE
为什么我设置了之后，只有标题栏没有生效？

### DISCUSSION THREAD

**Discussion by xxhhlk** [Maplespe/DWMBlurGlass#358] [Q&A] [answered]
配合ExplorerBlurMica，主窗体倒是有效果，但是标题栏没有，奇怪了，已经尝试切换浅色/深色，标题栏颜色启用/禁用，重启电脑。没有用其他的美化软件（MacType不算吧），下面是设置截图和效果截图，有没有大佬帮忙看看
![Snipaste_2024-06-30_22-40-29](https://github.com/Maplespe/DWMBlurGlass/assets/15075327/acec4758-9a24-4610-898b-139925da1083)
![Snipaste_2024-06-30_22-40-26](https://github.com/Maplespe/DWMBlurGlass/assets/15075327/9cd7901d-fdc6-4719-aa00-49e575abfe2a)
![Snipaste_2024-06-30_22-40-22](https://github.com/Maplespe/DWMBlurGlass/assets/15075327/28aa340e-fa0f-4acf-9055-278e0d0da4c6)
![Snipaste_2024-06-30_22-42-17_result](https://github.com/Maplespe/DWMBlurGlass/assets/15075327/ba82be2c-80b6-43d7-878e-c5062a9a4cff)
![Snipaste_2024-06-30_22-40-38](https://github.com/Maplespe/DWMBlurGlass/assets/15075327/7872469e-86ae-4fbb-adc7-7146788ddbf9)

**Comment by Maplespe** [comment]
卸载再安装 看看有没有错误提示

**Reply by xxhhlk** [reply]
已卸载后删除文件夹重新解压安装，没有错误提示，问题还是一样

**Comment by xxhhlk** [comment, marked as answer]
找到原因了，文件夹权限的原因，我放的是”下载“文件夹里面（这个文件夹搬到D盘了，所以可能软件没检测到），放到C盘某目录或者D盘其他目录之后都可以正常使用
